CRM/MembershipExtras/Hook/PostProcess/MembershipOfflineAutoRenewProcessor.php

```diff
@@ -72,8 +50,20 @@ private function isOfflineAutoRenewMembership() {
    *
    * @return mixed
    */
-  private function membershipIsAlreadyAutoRenew() {
-    return CRM_Utils_Request::retrieve('membership_is_already_autorenew', 'Int');
+  private function isMembershipAlreadyAutoRenewed() {
```

Comment:
The name of this function may lend itself for confusions, as it does not check if the membership has renewed automatically, as it implies. What it's really doing is checking if the membership is marked to auto-renew. So maybe we can change the method name to `isMembershipAlreadySetToAutoenew()`.